CMakeLists.txt

```diff
@@ -426,7 +426,7 @@ include_directories(SYSTEM ${MICROTAR_INCLUDE_DIR})
 
 set(MBXGEOM_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/third_party/geometry.hpp-0.9.2/include")
 include_directories(SYSTEM ${MBXGEOM_INCLUDE_DIR})
-set(CHEAPRULER_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/third_party/cheap-ruler-cpp-2.5.4/include")
+set(CHEAPRULER_INCLUDE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/third_party/cheap-ruler-cpp-2778eb8/include")
```

Comment:
Decided to update it to newer version. Mapbox doesn't set tags since 2017, but there was significant changes since that time.